### ISSUE TITLE
Pull Request to source code

### ISSUE THREAD

**Issue by mrkam2** [mamal72/open-in-code#2]
What I was looking for is an extension that allows to jump from the PR comment to the corresponding line of the code (existing in the local repo, without any downloads). Is this something you can add?

**Comment by mamal72** [comment]
Hi. Thanks for sharing your thoughts.

Hm... It seems a little tricky & not possible at the moment as the extension has no access to your local file system. It needs to know where you store your cloned projects to get access to them.
It's possible to implement download the repo and open that line in VSC but without downloading... Not now.
I think I can implement that too but it may take some time. It can be implemented by adding a settings page for the extension with a form for choosing root directory of cloned projects. This way, the extension can simply open that file in VSC. 🤔 
I've not enough time to work on this for now. I may do it in the future (maybe next few weeks).
Anyway, PRs are all welcome! 🙏 👍 😁

**Comment by mrkam2** [comment]
Yep, I was also thinking about settings page to set the root folder. 

Seems like another missing feature is to work on a single line vs. full page.

Thanks for considering this!
